### PR TITLE
Add “Help choose the syntax for CSS Nesting” post

### PR DESCRIPTION
**Pull request by bramus**
Apologies for not having a proposal or a draft here, but this is me shifting gears quickly in response to [webkit having published a post to ask for the CSS Nesting syntax](https://webkit.org/blog/13607/help-choose-from-options-for-css-nesting-syntax/).

Would be great to have this out so we can get people to vote before the voting window _(which is unknown)_ closes.

**Please proofread, as there has been no reviewed draft version of this post.**

### CODE REVIEW

File: site/en/blog/help-css-nesting-again/index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  The CSS Working Group is continuing a debate over the best way to define nesting in CSS. And if you are someone who writes CSS, we’d like your help.
+date: 2022-12-16
+tags:
+  - css
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/ZrMGmOcY6AH5J2tAblyO.jpg
+alt: >
+  A bird nest with a couple of blue eggs inside, surrounded by lush greenery.
+---
+
+After we ran [the previous survey to help choose the syntax for CSS Nesting](/blog/help-css-nesting), the CSS Working Group continued debate over the best way to define nesting in CSS. During the discussions new syntax ideas were suggested. To help the Working Group choose between any of these syntaxes, a new survey is launched.
```
Comment (rachelandrew) [inline site/en/blog/help-css-nesting-again/index.md:17]:
```suggestion
After we ran [the previous survey to help choose the syntax for CSS Nesting](/blog/help-css-nesting), the CSS Working Group continued debate over the best way to define nesting in CSS. During the discussions new syntax ideas were suggested. To help the Working Group choose between any of these syntaxes, there is a new survey.
```

File: site/en/blog/help-css-nesting-again/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+## The Options
+
+Based on [the results of the previous survey](/blog/help-css-nesting-survey), options 1 and 2 are longer considered. Instead two new options–numbered 4 and 5–were added.
```
Comment (rachelandrew) [inline site/en/blog/help-css-nesting-again/index.md:21]:
```suggestion
Based on [the results of the previous survey](/blog/help-css-nesting-survey), options one and two are longer being considered. Two new options (four and five) have been added.
```

File: site/en/blog/help-css-nesting-again/index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    property: value;
+  }
+}
+```
+
+While the snippets above are only used to demonstrate the basics of each proposal, there are many more examples included along with the new survey.
+
+## Cast your vote
+
+To cast your vote head over to [https://webkit.org/blog/13607/](https://webkit.org/blog/13607/). Voting is easy and requires no registration or anything like that. Simply choose “Option 5”, “Option 4”, or “Option 3” and hit Submit.
```
Comment (rachelandrew) [inline site/en/blog/help-css-nesting-again/index.md:69]:
```suggestion
To cast your vote head over to [https://webkit.org/blog/13607/](https://webkit.org/blog/13607/). Voting requires no registration, choose “Option 5”, “Option 4”, or “Option 3” and hit Submit.
```

File: site/en/blog/help-css-nesting-again/index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+---
+layout: "layouts/blog-post.njk"
+title: Help choose the syntax for CSS Nesting
+authors:
+  - bramus
+  - argyle
+description: >
+  The CSS Working Group is continuing a debate over the best way to define nesting in CSS. And if you are someone who writes CSS, we’d like your help.
```
Comment (rachelandrew) [inline site/en/blog/help-css-nesting-again/index.md:8]:
```suggestion
  The CSS Working Group is continuing a debate over the best way to define nesting in CSS. If you are someone who writes CSS, we’d like your help.
```

**Review by rachelandrew — APPROVED** [review]
Approving so I don't block as I'm mid travel, but please commit changes first

File: site/en/blog/help-css-nesting-again/index.md (hunk 2 above)
Comment (AaronForinton) [reply site/en/blog/help-css-nesting-again/index.md:21]: the `/blog/help-css-nesting-survey` link here is a 404 and I believe it should be `/blog/help-css-nesting-results/`

Comment (bramus) [reply site/en/blog/help-css-nesting-again/index.md]: Good catch @AaronForinton. Fixed in a later commit.

**Pull request merged**